### ISSUE TITLE
[Bug]: Bad image quality same model and settings vs normal webui

### ISSUE THREAD

**Issue by Dendane** [lllyasviel/stable-diffusion-webui-forge#85]
### Checklist

- [X] The issue exists after disabling all extensions
- [X] The issue exists on a clean installation of webui
- [ ] The issue is caused by an extension, but I believe it is caused by a bug in the webui
- [X] The issue exists in the current version of the webui
- [X] The issue has not been reported before recently
- [ ] The issue has been reported before but has not been fixed yet

### What happened?

When making a picture using t2i using AldobaseXL at 1024 x 1024 with 40 steps and cfg 7 and the same seed, the image quality is always worse in auto1111 forge compared to normal auto1111, see pictures in attachment, look at the skin and the eyes difference.

![sd](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/18679719/5c175a4f-5777-4026-845d-f6fd0d42cd52)
![sdforge](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/18679719/d8eec2c7-ca31-4aeb-8b07-55da9cb6cf83)

### Steps to reproduce the problem

Prompt: a close-up photograph off a German white woman wearing a German dress sitting together with a Senegalese woman wearing a Senegalese dress, colorful clothing, extremely detailed beautiful faces, extremely detailed fabric, high quality, 8k
-1024*1024
-cfg 7
-steps 40
-seed: 1506083485

### What should have happened?

show the same quality of image

### What browsers do you use to access the UI ?

Google Chrome

### Sysinfo

[sysinfo-2024-02-06-22-42.json](https://github.com/lllyasviel/stable-diffusion-webui-forge/files/14186796/sysinfo-2024-02-06-22-42.json)

### Console logs

```Shell
Creating venv in directory F:\AI\SD Forge\webui\venv using python "C:\Users\jona0\AppData\Local\Microsoft\WindowsApps\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\python.exe"
venv "F:\AI\SD Forge\webui\venv\Scripts\Python.exe"
Python 3.10.11 (tags/v3.10.11:7d4cc5a, Apr  5 2023, 00:38:17) [MSC v.1929 64 bit (AMD64)]
Version: f0.0.9-latest-52-gb58b0bd4
Commit hash: b58b0bd4259cf71077dfd7787fb77af4c02760a1
Installing torch and torchvision
Looking in indexes: https://pypi.org/simple, https://download.pytorch.org/whl/cu121
Collecting torch==2.1.2
  Using cached https://download.pytorch.org/whl/cu121/torch-2.1.2%2Bcu121-cp310-cp310-win_amd64.whl (2473.9 MB)
Collecting torchvision==0.16.2
  Using cached https://download.pytorch.org/whl/cu121/torchvision-0.16.2%2Bcu121-cp310-cp310-win_amd64.whl (5.6 MB)
Collecting fsspec
  Using cached fsspec-2024.2.0-py3-none-any.whl (170 kB)
Collecting sympy
  Using cached https://download.pytorch.org/whl/sympy-1.12-py3-none-any.whl (5.7 MB)
Collecting typing-extensions
  Using cached typing_extensions-4.9.0-py3-none-any.whl (32 kB)
Collecting jinja2
  Using cached Jinja2-3.1.3-py3-none-any.whl (133 kB)
Collecting networkx
  Using cached https://download.pytorch.org/whl/networkx-3.2.1-py3-none-any.whl (1.6 MB)
Collecting filelock
  Using cached filelock-3.13.1-py3-none-any.whl (11 kB)
Collecting numpy
  Using cached numpy-1.26.4-cp310-cp310-win_amd64.whl (15.8 MB)
Collecting requests
  Using cached requests-2.31.0-py3-none-any.whl (62 kB)
Collecting pillow!=8.3.*,>=5.3.0
  Using cached https://download.pytorch.org/whl/pillow-10.2.0-cp310-cp310-win_amd64.whl (2.6 MB)
Collecting MarkupSafe>=2.0
  Using cached MarkupSafe-2.1.5-cp310-cp310-win_amd64.whl (17 kB)
Collecting idna<4,>=2.5
  Using cached idna-3.6-py3-none-any.whl (61 kB)
Collecting charset-normalizer<4,>=2
  Using cached charset_normalizer-3.3.2-cp310-cp310-win_amd64.whl (100 kB)
Collecting urllib3<3,>=1.21.1
  Using cached urllib3-2.2.0-py3-none-any.whl (120 kB)
Collecting certifi>=2017.4.17
  Using cached certifi-2024.2.2-py3-none-any.whl (163 kB)
Collecting mpmath>=0.19
  Using cached https://download.pytorch.org/whl/mpmath-1.3.0-py3-none-any.whl (536 kB)
Installing collected packages: mpmath, urllib3, typing-extensions, sympy, pillow, numpy, networkx, MarkupSafe, idna, fsspec, filelock, charset-normalizer, certifi, requests, jinja2, torch, torchvision
Successfully installed MarkupSafe-2.1.5 certifi-2024.2.2 charset-normalizer-3.3.2 filelock-3.13.1 fsspec-2024.2.0 idna-3.6 jinja2-3.1.3 mpmath-1.3.0 networkx-3.2.1 numpy-1.26.4 pillow-10.2.0 requests-2.31.0 sympy-1.12 torch-2.1.2+cu121 torchvision-0.16.2+cu121 typing-extensions-4.9.0 urllib3-2.2.0

[notice] A new release of pip is available: 23.0.1 -> 24.0
[notice] To update, run: F:\AI\SD Forge\webui\venv\Scripts\python.exe -m pip install --upgrade pip
Installing clip
Installing open_clip
Installing requirements
Installing forge_legacy_preprocessor requirement: fvcore
Installing forge_legacy_preprocessor requirement: mediapipe
Installing forge_legacy_preprocessor requirement: onnxruntime
Installing forge_legacy_preprocessor requirement: svglib
Installing forge_legacy_preprocessor requirement: insightface
Installing forge_legacy_preprocessor requirement: handrefinerportable
Installing forge_legacy_preprocessor requirement: depth_anything
Launching Web UI with arguments:
Total VRAM 16379 MB, total RAM 16326 MB
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA GeForce RTX 4060 Ti : native
VAE dtype: torch.bfloat16
The cache for model files in Transformers v4.22.0 has been updated. Migrating your old cache. This is a one-time only operation. You can interrupt this and resume the migration later on by calling `transformers.utils.move_cache()`.
0it [00:00, ?it/s]
Using pytorch cross attention
ControlNet preprocessor location: F:\AI\SD Forge\webui\models\ControlNetPreprocessor
Loading weights [a928fee35b] from F:\AI\SD Forge\webui\models\Stable-diffusion\albedobaseXL_v20.safetensors
2024-02-06 23:38:26,826 - ControlNet - INFO - ControlNet UI callback registered.
Running on local URL:  http://127.0.0.1:7861

To create a public link, set `share=True` in `launch()`.
Startup time: 419.9s (prepare environment: 388.3s, import torch: 12.4s, import gradio: 5.0s, setup paths: 5.5s, initialize shared: 0.6s, other imports: 3.4s, setup gfpgan: 0.1s, load scripts: 2.7s, create ui: 0.9s, gradio launch: 0.8s).
model_type EPS
UNet ADM Dimension 2816
Using pytorch attention in VAE
Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
Using pytorch attention in VAE
extra {'cond_stage_model.clip_l.logit_scale', 'cond_stage_model.clip_g.transformer.text_model.embeddings.position_ids', 'cond_stage_model.clip_l.text_projection'}
loaded straight to GPU
To load target model SDXL
Begin to load 1 model
To load target model SDXLClipModel
Begin to load 1 model
Moving model(s) has taken 0.75 seconds
Model loaded in 28.9s (load weights from disk: 1.4s, forge load real models: 26.2s, calculate empty prompt: 1.3s).
100%|██████████████████████████████████████████████████████████████████████████████████| 40/40 [00:38<00:00,  1.04it/s]
To load target model AutoencoderKL█████████████████████████████████████████████████████| 40/40 [00:36<00:00,  1.11it/s]
Begin to load 1 model
Moving model(s) has taken 3.03 seconds
Total progress: 100%|██████████████████████████████████████████████████████████████████| 40/40 [00:41<00:00,  1.03s/it]
Total progress: 100%|██████████████████████████████████████████████████████████████████| 40/40 [00:41<00:00,  1.11it/s]
```

### Additional information

none

**Comment by Dendane** [comment]
sampling method: DPM++ SDE Karras

**Comment by lllyasviel** [comment]
thanks for the report.

can you change seed to 12345, using same settings and generate 4 images from both?

 result will decide the priority of the issue

**Comment by Dendane** [comment]
Forge pictures
![forge (1)](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/18679719/76f56275-f83a-4c57-9370-f10278ee6cba)
![forge (2)](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/18679719/e0f59bd4-a5db-4c4d-bc98-96145c6691cc)
![forge (3)](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/18679719/a612358b-fcda-4d17-81c7-f4487033a233)
![forge (4)](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/18679719/83135eff-5adb-4186-93f6-06665957ebce)

**Comment by Dendane** [comment]
normal webui pictures
![normal sd (1)](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/18679719/645a79b8-53e8-4ce3-98f2-8af3c641e93b)
![normal sd (2)](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/18679719/e425fc8b-cde5-466f-988b-383a6835dffc)
![normal sd (3)](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/18679719/82d837db-96d4-48af-ad68-81b88dde6965)
![normal sd (4)](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/18679719/08ae2fec-2f8a-4365-b16f-4d4e74cf18a8)

**Comment by Dendane** [comment]
![TEST](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/18679719/588a698b-598f-45d0-8376-d7f4b9df371e)

**Comment by lllyasviel** [comment]
Below is your meta in Automatic1111

a close-up photograph off a German white woman wearing a German dress sitting together with a Senegalese woman wearing a Senegalese dress, colorful clothing, extremely detailed beautiful faces, extremely detailed fabric, high quality, 8k
Steps: 40, Sampler: DPM++ SDE Karras, CFG scale: 7, Seed: 1506083485, **Face restoration: CodeFormer**, Size: 1024x1024, Model hash: a928fee35b, Model: albedobaseXL_v20, Version: v1.7.0

your meta in Forge

a close-up photograph off a German white woman wearing a German dress sitting together with a Senegalese woman wearing a Senegalese dress, colorful clothing, extremely detailed beautiful faces, extremely detailed fabric, high quality, 8k
Steps: 40, Sampler: DPM++ SDE Karras, CFG scale: 7, Seed: 1506083485, Size: 1024x1024, Model hash: a928fee35b, Model: albedobaseXL_v20, Version: f0.0.7-latest-41-g6aee7a20

so this issue is not valid and I am going to close this.

Feel free to open another one if other functionality has problems.
